examples/blender/blender.py

```diff
@@ -125,9 +128,18 @@ async def worker(ctx: WorkContext, tasks):
         f"yapapi version: {utils.TEXT_COLOR_YELLOW}{yapapi_version}{utils.TEXT_COLOR_DEFAULT}\n"
     )
     sys.stderr.write(f"Using subnet: {utils.TEXT_COLOR_YELLOW}{subnet}{utils.TEXT_COLOR_DEFAULT}\n")
```

Comment:
@azawlocki I'd also add the line listing a "network/driver" here for completeness sake

Reply:
Added in https://github.com/golemfactory/yapapi/pull/215/commits/456ae56b91d7bb588db8e41bed5e262a5476353e